csrankings-x.csv

```diff
@@ -221,6 +221,7 @@ Xiaoqing Zheng,Fudan University,http://homepage.fudan.edu.cn/zhengxq,jDNLWHIAAAA
 Xiaoqiu Huang 0001,Iowa State University,http://www.cs.iastate.edu/~xqhuang,h5gEfxsAAAAJ
 Xiaorong Pu,UESTC,https://www.en.scse.uestc.edu.cn/info/1085/1915.htm,cgue_KQAAAAJ
 Xiaoru Yuan,Peking University,http://vis.pku.edu.cn/xiaoruyuan.html,mJJVqRYAAAAJ
+Xiaorui Liu,North Carolina State University,https://sites.google.com/ncsu.edu/xiaorui/

```

Comment:
Missing Google Scholar entry.

Reply:
Added Google Scholar entry in a new commit. Thanks!